### DISCUSSION TITLE
WarpTerminal is not supportedFeat:

### DISCUSSION THREAD

**Discussion by ace-wangyuzhen** [aws/codewhisperer-command-line-discussions#97] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
> Please include a detailed description of the feature (and an image or screen recording, if applicable)

**Comment by ace-wangyuzhen** [comment]
<img width="930" alt="image" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/39297643/5c71e2bc-688c-4746-90ed-4c2bbcddc8be">

**Comment by grant0417** [comment]
Will link to the https://github.com/withfig/fig/issues/785, but basically it isn't possible for us to integrate with Warp due to them being a closed source terminal.
